Project_root/Components/Clock/Clock.css

```
@@ -0,0 +1,26 @@
+body {
+    background-color: orange;
+  }
+  #Block {
```

fix: no capital letters please
```suggestion
  #block {
```